### PR TITLE
auto repo specs ; repo/branch renaming

### PR DESCRIPTION
**Pull request by athornton**
WARNING: this should be merged immediately upon Simon's renaming repos and branches.  NOT before, and NOT long after.

### CODE REVIEW

File: services/nublado2/values-idfprod.yaml [hunk 1]
```diff
@@ -18,8 +18,10 @@ nublado2:
       PGPASSFILE: "/opt/lsst/software/jupyterlab/butler-secret/postgres-credentials.txt"
       AWS_SHARED_CREDENTIALS_FILE: "/opt/lsst/software/jupyterlab/butler-secret/aws-credentials.ini"
       S3_ENDPOINT_URL: "https://storage.googleapis.com"
-      # technical debt: for needed for DP0 because deployment branch is not parametrized
-      AUTO_REPO_URLS: https://github.com/SimonKrughoff/system-test,https://github.com/rubin-dp0/tutorial-notebooks
+      # technical debt: for needed for DP0 because deployment branch is
+      # not parametrized
+      AUTO_REPO_SPECS: https://github.com/lsst-sqre/system-test@prod,https://github.com/rubin-dp0/tutorial-notebooks@prod
```
Comment (SimonKrughoff) [inline services/nublado2/values-idfprod.yaml:23]: I think I'd prefer this in the same order as in the other values files just for consistency

**Review by SimonKrughoff — COMMENTED** [review]
I think that all jibes with what I understand

File: services/nublado2/values-idfprod.yaml (hunk 1 above)
Comment (athornton) [reply services/nublado2/values-idfprod.yaml]: Done.

**Review by SimonKrughoff — APPROVED** [review]
LGTM

**Pull request merged**